<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileEditWindow</class>
 <widget class="QDialog" name="TileEditWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Tiles</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/main16.png</normaloff>:/images/main16.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_Terrain">
       <property name="title">
        <string>Terrain</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="3">
         <widget class="QSlider" name="horizontalSlider_Height">
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QComboBox" name="comboBox_Terrain">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Height">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_Height">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Obstacles">
       <property name="title">
        <string>Obstacles</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QComboBox" name="comboBox_Obstacle">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Bumpers">
          <property name="maximumSize">
           <size>
            <width>44</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Bumpers</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkBox_North">
          <property name="text">
           <string>North</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBox_South">
          <property name="text">
           <string>South</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QCheckBox" name="checkBox_East">
          <property name="text">
           <string>East</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QCheckBox" name="checkBox_West">
          <property name="text">
           <string>West</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Rendering">
       <property name="title">
        <string>Rendering</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioButton_Layer1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Layer 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QRadioButton" name="radioButton_Layer2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Layer 2</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_Keep">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Keep</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox_Height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_Height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Height</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TileEditWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TileEditWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
